packaging/resource_suite_s3_nocache.py

```diff
@@ -2759,3 +2759,28 @@ def test_put_get_glacier_ir(self):
             s3plugin_lib.remove_if_exists(file2)
             s3plugin_lib.remove_if_exists(file1_get)
             s3plugin_lib.remove_if_exists(file2_get)
+
+class Test_S3_NoCache_MPU_Disabled_Base(Test_S3_NoCache_Base):
+
+    def test_iput_large_file_with_checksum_issue_2124(self):
+
+        try:
+
+            file1 = "issue_2124_file"
+            file1_get = "issue_2124_file.get"
+            file1_size = 32*1024*1024 + 1
```

Comment:
Maybe add a comment or even some words to the test name itself to explain why it needs to be 1 byte above 32MB?

Reply:
Well, partially it is because I don't remember if the cutoff for parallel transfer is >= 32 MB or > 32MB.

But I will put a comment in there.